data/eclipsefdn_projects.yaml

```diff
@@ -10,6 +10,7 @@
     tags: []
     github_repos: []
     gitlab_repos: []
+    gerrit_repos: []
```

Comment:
I never change this file, not sure why it shows up after I run npm install ... 🤔 Do I need to revert it?